test/lit/passes/optimize-instructions-bulk-memory.wast

```diff
@@ -1,5 +1,6 @@
 ;; NOTE: Assertions have been generated by update_lit_checks.py and should not be edited.
-;; RUN: wasm-opt %s --all-features --optimize-instructions --enable-bulk-memory -S -o - | filecheck %s
+;; RUN: wasm-opt %s --optimize-instructions --enable-bulk-memory --enable-simd -S -o - | filecheck %s
+;; RUN: wasm-opt %s --optimize-instructions --enable-bulk-memory -S -o - | filecheck %s --check-prefix NOSIMD
```

Comment:
I added a NOSIMD mode to avoid changing the existing results (which allowed SIMD as they had `--all-features`).